### ISSUE TITLE
Active Claims detail | Need help w/ Claim list item - Not readable with Talkback on Android (Accessibility)

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#17506]
## What happened?
Android TalkBack - Unable to single out this portion of the screen (Need help with claim) for Android to read. iOS is no problem.

## Specs:
- Device: Samsung Galaxy S7, talkback enabled.

## Steps to Reproduce
- Visit an active claim and try to read the 'Need help with claim' portion of the page with Talkback enabled.

## Desired behavior
All parts of the screen are read with screen reader software.

## Acceptance Criteria
- [ ] Talkback reads the "Need help with claim" portion of the screen.

## Bug Severity
- 4 - Low

## Linked Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16706

**Comment by digitalken** [comment]
PASS 1/12/2021 - Android now reads all sections of page.
